lib/Service/SettingsService.php

```diff
@@ -57,6 +57,12 @@ class SettingsService {
 	const DEFAULT_SENSITIVITY = '0.4';
 	const MAXIMUM_SENSITIVITY = '0.6';
 
+	/** Deviation used to suggestions */
+	const DEVIATION_KEY = 'deviation';
```

Comment:
maybe `cluster_deviation` to denote what this devaition is actually for?

Reply:
Use deviation as an analogy to the standard deviation in a probability, but certainly cannot be applied directly.. :sweat_smile: 
But I guess I can keep it.. Maybe cluster**s**_deviation